src/integrations/admin/social-templates-integration.php

```diff
@@ -164,6 +166,16 @@ protected function build_social_fields( Yoast_Form $yform, $identifier, $page_ty
 
 		\printf( '<div class="%s">', \esc_attr( $section_class ) );
 
+		echo '<div class="social-settings-heading-container">';
+		echo '<h3 class="social-settings-heading">' . \esc_html__( 'Social settings', 'wordpress-seo' ) . '</h3>';
+		if ( $is_premium ) {
+			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Is correctly escaped in the Premium_Badge_Presenter.
+			echo new Premium_Badge_Presenter( 'global-templates' );
```

Comment:
Not sure 'global-templates' is the right value since it's used to creater the id attribut of the badge, so we what it to be different for each one...

Reply:
Yep. the presenter adds a suffix so these IDs will result in:
- global-templates-premium-badge
- global-templates-new-badge

However, there are multiple instances of the badges in the same page so there will be duplicate IDs. Also, the home page tab uses the same ID for the "New" badge. Will need to make them unique.